components/resolved_ts/src/resolver.rs

```diff
@@ -142,7 +172,7 @@ impl Resolver {
             start_ts,
             self.region_id
         );
-        let bytes = key.as_slice().heap_size();
+        let bytes = self.lock_heap_size(&key);
```

Comment:
shall we calculate douple size here since `approximate_heap_bytes` has calculate both `locks_by_key` and `lock_ts_heap`?

Reply:
The majority of memory is consumed by keys, `locks_by_key` and `lock_ts_heap` shares the same `Arc<[u8]>`, so I think `lock_ts_heap` is negligible. Also, it's hard to track accurate memory use of `lock_ts_heap` as it's a `BTreeMap<TimeStamp, HashSet<Arc<[u8]>>>`, a timestamp may have many keys.